src/charmed_kubeflow_chisme/exceptions/_error_state_with_message.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+class ErrorStatusWithMessage(Exception):  # noqa: N818
```

Comment:
in Python world that will be read as: Error - Status with message

that is why you get N818 linter issue

and since it does not do any explicit thing to set the error state on the unit, may be any other more generic Error name ?

Reply:
overall idea is that it is our exception, we explicitly raise it. But it is nothing more than just named exception